starport_template/lib/stores/wallets_store.dart

```diff
@@ -50,4 +65,68 @@ class WalletsStore {
     wallets.value.add(creds.publicInfo);
     return creds.publicInfo;
   }
+
+  Future getCosmosBalances(BaseEnv baseEnv, String walletAddress) async {
```

Comment:
also, we're missing the Future's type here:
```suggestion
  Future<void> getCosmosBalances(BaseEnv baseEnv, String walletAddress) async {
```

Reply:
✅